helm_tests/other/test_keda.py

```diff
@@ -58,6 +58,17 @@ def test_keda_enabled(self, executor, is_created):
     @pytest.mark.parametrize(
         "executor", ["CeleryExecutor", "CeleryKubernetesExecutor", "CeleryExecutor,KubernetesExecutor"]
     )
+    def test_include_event_source_container_name_in_scaled_object(self, executor):
```

Comment:
Can you add a test for the triggerer [here](https://github.com/apache/airflow/blob/dabb6635070466c51490257d6b706107b9c775d3/helm_tests/airflow_core/test_triggerer.py#L756) as well?

Reply:
Sure! newer test is similar to the order, so it's name includes `for_triggerer` suffix. 
[link](https://github.com/apache/airflow/pull/44963/files#diff-9d8fcb21c167d5627a93575ced998738da9bd1b972830dd33daef4441424e773)